client/src/Movies/Movie.js

```diff
@@ -1,68 +1,55 @@
 import React, { Component } from 'react';
 import axios from 'axios';
+import MovieCard from './MovieCard';
 
 export default class Movie extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      movie: null
-    };
-  }
-
-  componentDidMount() {
-    // change this line to grab the id passed on the URL
-    const id = 1;
-    this.fetchMovie(id);
-  }
+    constructor(props) {
+        super(props);
+        this.state = {
+            movie : null
+        };
+    }
 
-  fetchMovie = id => {
-    axios
-      .get(`http://localhost:5000/api/movies/${id}`)
-      .then(response => {
-        this.setState(() => ({ movie: response.data }));
-      })
-      .catch(error => {
-        console.error(error);
-      });
-  };
-  // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
+    componentDidMount() {
+        // change this line to grab the id passed on the URL
+        const id = this.props.match.params.id;
```

Comment:
Why not use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here?
```javascript
const { id } = this.props.match.params;
```